src/main/java/org/broadinstitute/barclay/argparser/NamedArgumentDefinition.java

```diff
@@ -131,12 +128,6 @@ public NamedArgumentDefinition(
      */
     public boolean isAdvanced() { return getUnderlyingField().getAnnotation(Advanced.class) != null; }
 
-    /**
-     * return true if this argument has the {@code @Deprecated} annotation.
-     * @return true if this argument is advanced, otherwise false.
-     */
-    public boolean isDeprecated() { return getUnderlyingField().isAnnotationPresent(Deprecated.class); }
```

Comment:
So previously this would be triggered by the standard java `Deprecated` annotation?  Should we continue to respect those as well?  I don't remember why exactly we decided to use a separate annotation instead of the default one.  So that intellij doesn't tag uses of it?

Reply:
The old code was carried over from the GATK3 implementation, but never fully embraced, and only worked for args IIRC (not tools or other documented features). I think we used a custom annotation both for the IntelliJ issue, and also because we wanted to be able to specify a reason/detail string, which the Java annotation doesn't have. There are a couple of usages of the old tag in GATK for arguments, but I'd prefer to update them rather than carry around (and test) both annotations.